File: code_examples/nightingale.py
ACTIVE_TIME = 'Night (10 p.m. to 7 a.m.)'

class Nightingale():
    name = 'Common Nightingale'
    scientific_name = 'Luscinia megarhynchos'
    population_status = 'stable'
    average_mass = 20#g
    active_time = ACTIVE_TIME
    
    def __str__(self):
        return f'Name: {self.name}'
    
    def sing(self):
        for i in range(10):
            print('Coding...')

if __name__=='__main__':
    nightingale = Nightingale()
    nightingale.sing()

File: code_examples/nightingale.cpp
#include <iostream>
using namespace std;

const string ACTIVE_TIME = "Night (10 p.m. to 7 a.m.)";

class Nightingale
{
public:
    string name = "Common Nightingale";
    string scientific_name = "Luscinia megarhynchos";
    string population_status = "stable";
    int average_mass = 20; // g
    string active_time = ACTIVE_TIME;

public:
    void printDetails()
    {
        cout << "Name : " << name << endl;
        cout << "Scientific Name : " << scientific_name << endl;
        cout << "Average Mass : " << average_mass << 'g' << endl;
        cout << "Population Status : " << population_status << endl;
        cout << "Active Time : " << active_time << endl;
    }
    
    void sing()
    {
        for (int i = 0; i < 10; i++)
        {
            cout << "Coding..." << endl;
        }
    }
};

int main()
{
    Nightingale nightingale;
    nightingale.sing();
    nightingale.printDetails();
}



